### DISCUSSION TITLE
Whats going on here?

### DISCUSSION THREAD

**Discussion by bashar0151** [mastodon/mastodon#25814] [General]
Hello world.

I thought about doing a project today to make a mastodon server, Followed some guides and manged to get it running with no errors until .... https://scouse.page/ Web page only dispalys the Mastodon logo. 

OS: Ubuntu  20.4  (Fresh Install)
Steps taken:[]( https://www.linuxbabe.com/ubuntu/how-to-install-mastodon-on-ubuntu)
System Spec: 2 CPU 4GB Ram 80GB storage fresh install
Mastodon version: 4.14 (Latest)

---
mastodon-web  log
Jul 07 22:23:23 scouse.page bundle[13749]: [ActionDispatch::HostAuthorization::DefaultResponseApp] Blocked host: www.universitynewshq.com
Jul 07 22:24:33 scouse.page bundle[13749]: [ActionDispatch::HostAuthorization::DefaultResponseApp] Blocked host: universitynewshq.com
Jul 07 22:28:27 scouse.page bundle[13736]: [ActionDispatch::HostAuthorization::DefaultResponseApp] Blocked host: whiterabbitesla.com
Jul 07 22:28:27 scouse.page bundle[13749]: [ActionDispatch::HostAuthorization::DefaultResponseApp] Blocked host: whiterabbitesla.com
Jul 07 22:28:53 scouse.page bundle[13736]: [ActionDispatch::HostAuthorization::DefaultResponseApp] Blocked host: www.rtpmekar4d.art
~

---

Any suggestions on how this can be fixed?

**Comment by renchap** [comment]
Your server seems to be misconfigured, JS and CSS files are not loading properly.

I get those errors in the console:

```
[Error] Failed to load resource: the server responded with a status of 404 () (application-4b09753ec8a1c928c4ba.chunk.js, line 0)
[Error] Failed to load resource: the server responded with a status of 404 () (locale_en-GB-327345b3d4a784211ba6.chunk.js, line 0)
[Error] Failed to load resource: the server responded with a status of 404 () (default-a3b88036.chunk.css, line 0)
[Error] Failed to load resource: the server responded with a status of 404 () (common-a844dc34.css, line 0)
[Error] Failed to load resource: the server responded with a status of 404 () (common-0173e2f60b9cdf71cbe2.js, line 0)

[Error] Refused to execute https://scouse.page/packs/js/common-0173e2f60b9cdf71cbe2.js as script because "X-Content-Type-Options: nosniff" was given and its Content-Type is not a script MIME type.
[Error] Refused to execute https://scouse.page/packs/js/locale_en-GB-327345b3d4a784211ba6.chunk.js as script because "X-Content-Type-Options: nosniff" was given and its Content-Type is not a script MIME type.
[Error] Refused to execute https://scouse.page/packs/js/application-4b09753ec8a1c928c4ba.chunk.js as script because "X-Content-Type-Options: nosniff" was given and its Content-Type is not a script MIME type.
```

`https://scouse.page/packs/js/common-0173e2f60b9cdf71cbe2.js` for example is returning a 404 error, which is not normal. Those files should be served by nginx if you are using the config from the docs.
